src/System.Net.Http/src/System/Net/Http/Managed/HttpSystemProxy.cs

```diff
@@ -12,25 +12,27 @@
 
 namespace System.Net.Http
 {
-    internal sealed class HttpSystemProxy : IWebProxy
+    internal sealed class HttpSystemProxy : IWebProxy, IDisposable
     {
         private readonly Uri _proxyUri;         // String URI for HTTPS requests
-        private string[] _bypass = null;        // list of domains not to proxy
+        private string[] _bypass;               // list of domains not to proxy
```

Comment:
nit: looks like an extra tab character after semicolon.

Reply:
it may be some editor setting. But I don't see it in hexdump.

```
00000250  73 74 72 69 6e 67 5b 5d  20 5f 62 79 70 61 73 73  |string[] _bypass|
00000260  3b 20 20 20 20 20 20 20  20 20 20 20 20 20 20 20  |;               |
00000270  2f 2f 20 6c 69 73 74 20  6f 66 20 64 6f 6d 61 69  |// list of domai|
```

Is there any recommended setting for Visual Studio?